admin-dev/themes/new-theme/.webpack/dev.js

```diff
@@ -11,10 +16,68 @@ function devConfig() {
     {
       devtool: 'inline-source-map',
       devServer: {
+        client: {
+          logging: 'error',
+          progress: false,
+          overlay: {
+            errors: true,
+            warnings: false,
+          },
+        },
         hot: true,
-        contentBase: path.resolve(__dirname, '/../public'),
-        publicPath: '/',
+        static: {
+          directory: path.join(__dirname, '/../public'),
+          watch: false,
+        },
+        port: 3000,
+        watchFiles: [
+          path.join(__dirname, '/../**/*.tpl'),
+          path.join(__dirname, '../js/**/*.js'),
+          path.join(__dirname, '../scss/**/*.scss'),
+        ],
+        open: true,
+        proxy: {
+          '**': {
+            target: process.env.PS_URL,
```

Comment:
Is it possible to have a way to configure this as well? So that we don't need to rely only on ENV var

Because, personally, copying my local URL all the time just to run this will pretty soon become a pain, so I'll stop using the feature 😅  We could have a config file that is used as an alternative, this way once it is configured we could simply run `npm run serve` and that's it

What do you think?

Reply:
We could add it do a JS file that we ignore? 

Or you can configure it inside your own environment.

I don't feel like i's a pain to me since the command is autocomplete inside my shell, but as you want